js/samples/06.auth.botSSO/README.md

```diff
@@ -0,0 +1,156 @@
+# Teams SSO Bot
+
+This sample shows how to incorporate a basic conversational SSO flow into a Microsoft Teams application using [Bot Framework](https://dev.botframework.com) and the Teams AI SDK.
```

Comment:
How should the user interact with the bot, and what should they expect? What are the pre-reqs for getting auth to work?

Reply:
Will be fixed in another ticket: #783